### ISSUE TITLE
Trying to python my way to API data.

### ISSUE THREAD

**Issue by JacksonSlade** [Kyria/EsiPy#47]
I am trying to follow the instructions here:

https://kyria.github.io/EsiPy/examples/sso_login_esipy/

Invariably, I get the error in screen shot 1 👍 

![screen shot 1](https://user-images.githubusercontent.com/43481266/45901016-71bdf100-bdaf-11e8-829f-04dbc61b5c97.PNG)

I am using the script correctly, as I get a url that makes me log into the eve online system for authentication:

![screen shot 2](https://user-images.githubusercontent.com/43481266/45901067-a2058f80-bdaf-11e8-8431-adf15fe256c3.PNG)

I use the url in a browser, then copy out the code from the call back to use in the auth function/object:

print(tokens = security.auth('5kw    ...   S2ZA'))

and it errors to that error from screen shot 1.  

I am on windows 10, using python 3.6.  I replicated this error on a mac, also using pythong 3.6.  I follow the instructions and the token never gets built.  What am I doing wrong?

**Comment by Kyria** [comment]
As stated here: https://github.com/Kyria/EsiPy/issues/46 you should remove app parameter from security, or manually set sso_uri to the v2 auth. 

I've started to make the change but i didn't have the time to finalize and release it :) (btw i'll need to change examples too)

**Comment by JacksonSlade** [comment]
In Security.py, line 55 - edit the line to read:
sso_url = kwargs.pop('sso_url', "https://login.eveonline.com/v2")
In security.py, line 84 - edit the line to read:
self.oauth_token = '%s://%s/v2/oauth/token' %  (
In security.py, line 88 - edit the line to read:
self.oauth_revoke = '%s://%s/v2/oauth/revoke' %

Not doing your job, Just posting this for others who find there way here.  Doing this resulted in python returing a token with security.auth called with the code from a browser

**Comment by Kyria** [comment]
Actually, this will break when i'll update esipy :) 
The correct way to do this is just to override sso_url when instanciating the EsiSecurity object, **without** giving it "app" parameter.
```python
secu = EsiSecurity(sso_url="https://login.eveonline.com/v2")
```
and you are done (without the risk of breaking everything)

**Comment by Kyria** [comment]
Released in v1.0.0, please have a look at the release note and the docs :)
